src/providers/ShipSnapshotProvider/ShipSnapshotProvider.tsx

```diff
@@ -189,6 +195,47 @@ export const ShipSnapshotProvider = (props: ShipSnapshotProps) => {
     });
   }, []);
 
+  const moveModule = React.useCallback((fromFlag: number, toFlag: number) => {
+    setCurrentFit((oldFit: EsiFit | undefined) => {
+      if (oldFit === undefined) return undefined;
+
+      const newItems = [...oldFit.items];
+
+      const fromItemIndex = newItems.findIndex((item) => item.flag === fromFlag);
+      const fromItem = newItems[fromItemIndex];
+
+      const toItemIndex = newItems.findIndex((item) => item.flag === toFlag);
+      const toItem = newItems[toItemIndex];
+
+      fromItem.flag = toFlag;
+
+      if (toItem !== undefined) {
+        /* Target slot is non-empty, swap items */
```

Comment:
```suggestion
        /* Target slot is non-empty, swap items. */
```

The nitpick of all nitpicks ;)